src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -164,6 +164,11 @@
   <data name="MisplacedTransformFileWarning" xml:space="preserve">
     <value>The transform file '{0}' is outside the 'content' folder and hence will not be transformed during installation of this package. Move it into the 'content' folder.</value>
   </data>
+  <data name="NoRefOrLibFolderInPackage" xml:space="preserve">
+    <value>This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all
+frameworks. Since framework specific files were found under the build/ directory for {0}, consider creating the following empty files to correctly narrow the compatibility of the package:
```

Comment:
Why is there a new line in the string? How does this affect the output message in terminals of different widths? In particular why is the second line much longer than the first line?